pallets/backed_asset/README.md

```diff
@@ -1,3 +1,30 @@
 # Backed Asset Pallet
 
-An asset that is backed by one or more similar assets keeping track of the share that each collateral holds.
+This pallet allows creation of asset that is backed by one or more similar assets keeping track of the share that each collateral holds. Any liquidity provider can create a backed asset by depositing collateral to this pallet.
+
+## Terminology
+
+- Liquidity Provider : Any external user/company that provides liquidity to VLN by depositing up permitted collateral to mint assets on VLN.
```

Comment:
I just liquidity provider here instead of attestor in the other pallet. Is there a better name?

Reply:
Sounds good 👍🏽